modules/core/client/directives/page-title.client.directive.js

```diff
@@ -19,13 +19,17 @@
 
       function listener(event, toState) {
         var applicationCoreTitle = 'MEAN.js',
-          separeteBy = ' - ';
+          separator = ' - ',
+          stateTitle = applicationCoreTitle + separator;
+
+        toState.name.split('.').forEach(function(value, index) {
+          stateTitle = stateTitle + value.charAt(0).toUpperCase() + value.slice(1) + separator;
+        });
         if (toState.data && toState.data.pageTitle) {
-          var stateTitle = $interpolate(toState.data.pageTitle)($state.$current.locals.globals);
-          element.html(applicationCoreTitle + separeteBy + stateTitle);
-        } else {
-          element.html(applicationCoreTitle);
+          stateTitle = $interpolate(stateTitle + toState.data.pageTitle + separator)(($state.$current.locals.globals));
```

Comment:
I misunderstood these latest commits, to be that you were using `pageTitle` as an override to using the State name as the resulting Page Title. I sort of liked that idea. Does this sound like it would be better than using a combination of State name, and `pageTitle`? For instance, a user could explicitly specify a Title using the `pageTitle` option in the route config to bypass the use of the State name. Otherwise, if the option isn't specified then the State name would be used.

I don't see much benefit in using both at the same time.